ckanext/canada/templates/internal/package/wet_datatable.html

```diff
@@ -44,3 +22,64 @@ <h2 id="prtable">{{_("Preview")}}</h2>
     </tbody>
   </table>
 </section>
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.20/css/jquery.dataTables.min.css"/>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/select/1.3.1/css/select.dataTables.min.css"/>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/buttons/1.6.1/css/buttons.dataTables.min.css"/>
+
+<script type="text/javascript" src="https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript" src="https://cdn.datatables.net/select/1.3.1/js/dataTables.select.min.js"></script>
+<script type="text/javascript" src="https://cdn.datatables.net/buttons/1.6.1/js/dataTables.buttons.min.js"></script>
+<script type="text/javascript" src="https://cdn.datatables.net/buttons/1.6.1/js/buttons.colVis.min.js"></script>
+<script type="text/javascript">
+
+
+  $( '#dtprv' ).DataTable({
+    "paging": true,
+    "serverSide": true,
+    "processing": true,
+    "ajax": {
+      "url": "{{ h.url_for(
+      controller='ckanext.canada.controller:CanadaController',
+      action='datatable',
+      resource_name=resource_name,
+      resource_id=resource_id,
+  ) }}",
+  "type": "POST"
+  },
+  select:{style: 'multi'},
+  "order": [[1, "asc"]],
+          columns: [
+    {"className":"select-checkbox","orderable":false,"targets":0},
+    {% if chromo.edit_form %}
+  {"className": "bg-info", "orderable": false, "targets": "no-sort"},
+  {% endif %}
+  {% for f in chromo.fields %}
+  {{- "null" if 'preview_class' not in f else
+    '{"className":"' | safe ~ f['preview_class'] ~ '"}' | safe }}
+  {%- if not loop.last %},{% endif %}
+  {% endfor %}],
+  dom: "Blfrtip",
+          buttons: [
+    'colvis',
+    {extend:"selected",text:"Edit in template",action: function ( e, dt, button, config ){
+        var row_data = JSON.stringify(dt.rows({selected:true}).data().toArray());
```

Comment:
as I mentioned on the other repo, instead of JSON.stringify you can just repeat the <input> fields with the same name to POST multiple primary key values to the recombinant endpoint